### PR TITLE
#1061 add action to check if PR's commits are signed

### LINKED ISSUE #1061: CI Check for signed commits
**What Needs to be Done?**

Add an action to do a CI check that all commits are properly signed, so that we can see when this has failed before all other tests have passed

**Is your feature request related to a problem? Please describe.**

**Describe potential solution outcome**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by jstrzebonski**
Fixes #1061

### CODE REVIEW

**Comment by jstrzebonski** [comment]
Seems to be working. If I push an unsigned commit, check returns an error.

File: .github/workflows/check-commits-are-signed.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+on:
+  pull_request
+
+jobs:
+  check:
+    name: Check if there are unsigned commits
+    runs-on: ubuntu-latest
+    steps:
+      - uses: DARMA-tasking/find-unsigned-commits@1061-check-for-signed-commits
```
Comment (jstrzebonski) [inline .github/workflows/check-commits-are-signed.yml:11]: Will be `@master` after merging DARMA-tasking/find-unsigned-commits#1

Comment (lifflander) [reply .github/workflows/check-commits-are-signed.yml:11]: It's merged if you want to switch this to master

**Review by lifflander — APPROVED** [review]
Good to go as soon as it switches to master.

**Comment by jstrzebonski** [comment]
> 
> 
> Good to go as soon as it switches to master.

Done. And also rebased.
Will merge this as soon as all tests pass.

**Pull request merged**